documentation/my-homeworks/week3.2/src/components/app.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import CreateTodo from './create-todo';
+import TodosList from './todos-list';
+import todos from './todos.json'
+
+
+export default class App extends React.Component {
+    state = {
+            myTodos:todos
```

Comment:
You named your state key as `myTodos` but you are using it as `this.state.todos` in most of your code.